Docker-compose/docker-compose.yml

```diff
@@ -18,66 +17,50 @@ services:
       # - mongodata:/data/db # to be used for external volumes
     ports:
       - '27017-27019:27017-27019'
-    networks:
-      - iam-proxy-italia
 
-  satosa-mongo-express:
+  express:
     profiles:
       - demo
       - mongoexpress
     image: mongo-express
-    container_name: satosa-mongo-express
     ports:
       - 8082:8081
     depends_on:
-      - satosa-mongo
+      - mongo
     environment:
       ME_CONFIG_BASICAUTH_USERNAME: "${EXPRESS_USER:-satosauser}"
       ME_CONFIG_BASICAUTH_PASSWORD: "${EXPRESS_PASSWORD:-satosapw}"
       ME_CONFIG_MONGODB_ADMINUSERNAME: "${MONGO_DBUSER:-satosa}"
       ME_CONFIG_MONGODB_ADMINPASSWORD: "${MONGO_DBPASSWORD:-thatpassword}"
       ME_CONFIG_MONGODB_URL: mongodb://${MONGO_DBUSER:-satosa}:${MONGO_DBPASSWORD:-thatpassword}@satosa-mongo:27017/
       TZ: "${TZ:-Europe/Rome}"
-    networks:
-      - iam-proxy-italia
 
-  django_sp:
+  sp:
```

Comment:
this is a django sp, therefore its name is django sp

Reply:
If we haven't other SP, why distinguish them?
But if you prefer we can call this django-sp :smile: 
Remember: the container name will be iam-proxy-italia-django-sp-1
With a long project name a short service is useful